udata/core/dataset/models.py

```diff
@@ -150,6 +152,8 @@ def is_available(self):
 
 
 class Resource(ResourceMixin, WithMetrics, db.EmbeddedDocument):
+    ''' A Resource is a resource provided by the original provider of the
+    dataset'''
```

Comment:
It's repeating itself: "a resource is a resource".
I'l change for "a resource is a file (remote or locale) or an API provided by..."